pages/cars.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <li v-for="i in list">{{ i }}</li>
```

Comment:
See how to use `v-for` [here](https://vuejs.org/api/built-in-directives.html#v-for)
You need to add `:key`

Reply:
Okay